apollo-integration/src/test/java/com/apollographql/apollo/ApolloWatcherTest.java

```diff
@@ -34,13 +33,11 @@
 public class ApolloWatcherTest {
   private ApolloClient apolloClient;
   private MockWebServer server;
-  private NormalizedCache normalizedCache;
-  private static final int TIME_OUT_SECONDS = 3;
+  private static final int TIME_OUT_SECONDS = 5;
```

Comment:
this is my least favorite change in this pr.  I don't think the issue is that our code takes an addl 2 seconds to run on a jvm